### PR TITLE
Psychological updates for usability testing

### PR DESCRIPTION
**Pull request by floresj**
Issues: 
- #1030 Review screen (and form wide)
- #1044 Why we are asking - text duplicated (#1064 duplicate)
- #1051 Admission voluntary/involuntary
- #1039 Text button style
- #1037 Headline margins
- #1028 Need margin between accordion title and date
- #1032 closed accordion style and content tweaks
- #1034 Closed accordion sections title tweaks
- #1010 input issue
- #1040 Diagnosed - text layout

### CODE REVIEW

File: src/components/Section/Psychological/Competence/Competence.jsx [hunk 1]
```diff
@@ -66,13 +62,16 @@ export default class Competence extends ValidationElement {
...
     return (
-      <span>
-        <span className="index">{type}</span>
-        <span className="info"><strong>{courtName}</strong></span>
+      <span className="content">
+        <span className="index">{type} {index + 1}:</span>
+        <span className="courtname">
+          <strong>{courtName || i18n.t('psychological.competence.collection.summaryCourtName')}</strong>
+        </span>
+        <span className="occurred"><strong>{courtName && occurred}</strong></span>
```
Comment (bmallred) [inline src/components/Section/Psychological/Competence/Competence.jsx]: Does this just return a boolean value?

Comment (floresj) [inline src/components/Section/Psychological/Competence/Competence.jsx]: No. Renders the value of occurred if courtName has a value.

File: src/components/Section/Psychological/Order.jsx [hunk 2]
```diff
@@ -84,20 +78,11 @@ export default class Order extends ValidationElement {
...
-                onUpdate={this.updateDisposition}
-                onValidate={this.props.onValidate}
-                />
-        </Field>
-
         <BranchCollection
           className="appeals"
           branchHelp={`psychological.${prefix}.help.appealed`}
           branch={<h3>{ i18n.t(`psychological.${prefix}.heading.appealed`) }</h3>}
+          branchTail={<h3>{ i18n.t(`psychological.${prefix}.heading.appealedAnother`) }</h3>}
```
Comment (bmallred) [inline src/components/Section/Psychological/Order.jsx]: Just a note to **self**... Change this to coincide with recent `BranchCollection` changes

Comment (floresj) [inline src/components/Section/Psychological/Order.jsx]: :+1: :+1:

**Pull request merged**